db/migrate/20210425220908_drop_create_friendly_id_slugs.rb

```diff
@@ -0,0 +1,5 @@
+class DropCreateFriendlyIdSlugs < ActiveRecord::Migration[6.1]
+  def change
+    drop_table :friendly_id_slugs
+  end
+end
```

Comment:
You do not need this migration. Since, `friendly_id_slugs` table is not merged in master branch, fell free to rollback the  `db/migrate/20210422194416_create_friendly_id_slugs.rb` migration and remove it.

Reply:
Thanks, done)